Added @family filter to listFilterMethods (issue #1676)

AlexEngelhardt opened the pull request:
Just a one-liner added to Filter.R (and five respective .Rd files). 
This is my first pull request ever, so I'm happy to receive hints if I did something wrong.
Closes #1676

giuseppec commented:
We never push the. Rd files as they are auto-generated by Travis anyway. But should be OK in this case. Did you find other "filter" functions where adding @family would also make sense?

AlexEngelhardt commented:
Oh, alright.
There are two other functions with 'filter' in their name and no '@family filter' in them: makeFilter and getBMRFilteredFeatures. Should I add them as well?

AlexEngelhardt commented:
Three commits: 
- I deleted the .Rd modifications again
- I added "@family filter" to listFilterMethods() and now additionally to makeFilter()
- An update merge from your master
